.github/helper/install.sh

```diff
@@ -23,6 +23,7 @@ if [ "$DB" == "mariadb" ]
 then
   mariadb --host 127.0.0.1 --port 3306 -u root -ptravis -e "SET GLOBAL character_set_server = 'utf8mb4'";
   mariadb --host 127.0.0.1 --port 3306 -u root -ptravis -e "SET GLOBAL collation_server = 'utf8mb4_unicode_ci'";
+  mariadb --host 127.0.0.1 --port 3306 -u root -ptravis -e "SET GLOBAL innodb_compression_default = ON";
```

Comment:
Some notes:

1. We can't enable this globally
2. We already have a lot of sites where we might want to enable this. 

This should be done via `transform-database` command to optionally enable/disable on existing sites. 

For new one if this works we can try it on per table basis. 

Also this feature requires file per table option, so if it's not enabled enabling this won't do anything. 

---

Also when I tried this I noticed no difference, lets first verify on reasonably large DB with `DYNAMIC` row type to see if we do get some benefit. Most likely I missed SOME configuration step which prevented me from seeing any difference.

Reply:
Hmm based on my understanding, with the current defaults + enabling this, all _new_ tables should use it by default atleast.

Will test out a bit here as well and see if I notice any differences in storage used.